File: Solutions/2027/2027.c
/***************************************************************************************************
 * Copyright: washing
 * FileName: 2027.c
 * Author: washing
 * Version: 1.0
 * Date: 2022-12-27 10:01:28
***************************************************************************************************/


int minimumMoves(char * s){
    int ret=0;
    for (int i=0; i<strlen(s); i++) {
        if (s[i] == 'X') {
            s[i] = 'O';
            if (i+1 < strlen(s)) s[i+1] = 'O';
            if (i+2 < strlen(s)) s[i+2] = 'O';
            ret++;
        }
    }
    return ret;
}

File: Solutions/2027/2027.py
# -*- coding:utf-8 -*-
# Author: washing
# DateTime: 2022/12/27 10:16
# File: 2027.py
# Desc: 


class Solution:
    def minimumMoves(self, s: str) -> int:
        ret = 0
        l = list(s)
        for i in range(len(l)):
            if l[i] == "X":
                l[i:i+3] = ['O','O','O']
                ret += 1
        return ret
